docs/extend/modular-resources/create-subtype.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Define a New Resource Subtype"
+linkTitle: "New API Subtype"
+weight: 30
+type: "docs"
+tags: ["rdk", "extending viam", "modular resources", "API"]
+description: "Define an API for a new type of resource."
+no_list: true
+---
+
+You need to define a new resource *subtype* (and an API for that subtype) if:
+
+- You have a {{% glossary_tooltip term_id="resource" text="resource" %}} that does not fit into any of the existing [component](/components/) or [service](/services/) subtypes.
```

Comment:
I think you may be able to clarify what "fit into" means technically

Reply:
```suggestion
- You have a {{% glossary_tooltip term_id="resource" text="resource" %}} that does not fall under any of the existing [component](/components/) or [service](/services/) subtypes.
```
I realize this isn't any more technical, but is it better? The thing is, we could say "You have a resource that requires an API that is different from any of the existing resource APIs." but that would encompass both this and the following bullet into one. The point I am trying to make with these two bullets is that whether or not your thing is an arm/camera/vision service/whatever other existing resource subtype, you might want a different API, and I think that's more clear as these two bullets rather than one, but maybe my intuition is wrong there.